src/test/java/edu/cornell/kfs/kns/datadictionary/validation/fieldlevel/CuEmailAddressValidationPatternTest.java

```diff
@@ -0,0 +1,80 @@
+package edu.cornell.kfs.kns.datadictionary.validation.fieldlevel;
+
+import org.junit.Test;
+import org.kuali.kfs.kns.datadictionary.validation.fieldlevel.EmailAddressValidationPattern;
+import org.powermock.api.mockito.PowerMockito;
+
+import edu.cornell.kfs.gl.service.impl.fixture.EmailAdressTestValue;
+
```

Comment:
Test case will need to be adjusted for enum typos.

Reply:
ok